middleware/index.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
added a couple eslint appeasements along the way to try to reduce the amount of "red" in VScode